### ISSUE TITLE
Separate out debug and non-debug versions of installer packages?

### ISSUE THREAD

**Issue by stephengtuggy** [vegastrike/Vega-Strike-Engine-Source#573]

**Comment by BenjamenMeyer** [comment]
Possibly in the long term; but we probably have enough going on right now that we shouldn't do this quite yet.
Perhaps a short term task for this could be adding a flag to CMake that would allow folks to strip the symbols if they chose; by default it would be off (all symbols would remain); but distributors can then easily choose to strip the symbols for their distros.

**Comment by stephengtuggy** [comment]
> Perhaps a short term task for this could be adding a flag to CMake that would allow folks to strip the symbols if they chose...

Well, as far as that goes, folks can simply build in Release configuration, and it won't include debug symbols.

**Comment by stephengtuggy** [comment]
It looks like [this CMake 3.13 feature](https://cmake.org/cmake/help/latest/cpack_gen/deb.html#packaging-of-debug-information) would make things easy.

**Comment by stephengtuggy** [comment]
We could also use [this](https://cmake.org/cmake/help/latest/cpack_gen/rpm.html#packaging-of-debug-information) on the RPM side.

**Comment by BenjamenMeyer** [comment]
@stephengtuggy  yes, those look useful. We should experiment with them to see how it works, what it produces, etc.

**Comment by stephengtuggy** [comment]
@BenjamenMeyer sounds good.

**Comment by BenjamenMeyer** [comment]
We typically build with Release-Debug so that we get the release optimizations but can still capture stuff for a debugger. Do we really need this still?

**Comment by stephengtuggy** [comment]
> We typically build with Release-Debug so that we get the release optimizations but can still capture stuff for a debugger. Do we really need this still?

You mean `RelWithDebInfo` ?
